ci/Jenkinsfile-ppc64le-cuda8

```diff
@@ -28,7 +28,7 @@ def labelruntime = "ibm-power || ibm-power-gpu"
 def doingbenchmark = "0"
 def dobenchmark = "0"
 def doruntime = "1"
-def data_dirs = "-v /home/jenkins/smalldata:/smalldata -v /home/jenkins/open_data:/open_data"
+def data_dirs = "-v /home/0xdiag/h2o4gpu/data:/data -v /home/0xdiag/h2o4gpu/open_data:/open_data"
```

Comment:
Great, so jenkins ppc setup was just wrong for the data path?  Same path as x86?

Reply:
Yes and Yes.

Also sync jobs were not running.